charts/kubernetes-external-secrets/crds/kubernetes-client.io_externalsecrets_crd.yaml

```diff
@@ -8,73 +8,75 @@ metadata:
     app.kubernetes.io/managed-by: helm
 spec:
   group: kubernetes-client.io
-  version: v1
   scope: Namespaced
 
-  names:
-    shortNames:
-      - es
-    kind: ExternalSecret
-    plural: externalsecrets
-    singular: externalsecret
-
-  additionalPrinterColumns:
-    - JSONPath: .status.lastSync
-      name: Last Sync
-      type: date
-    - JSONPath: .status.status
-      name: status
-      type: string
-    - JSONPath: .metadata.creationTimestamp
-      name: Age
-      type: date
+  preserveUnknownFields: false
 
-  validation:
-    openAPIV3Schema:
-      required:
-        - spec
-      properties:
-        spec:
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      subresources:
+        status: {}
+      schema:
+        openAPIV3Schema:
+          required:
+            - spec
           type: object
+          required:
+            - spec
```

Comment:
The `required` field is defined twice (that's why the e2e test fails)

Reply:
Guess I failed rebase after all 😸 ill see if I get some time to incorporate latest additions as well